audbackend/core/backend.py

```diff
@@ -592,13 +691,35 @@ def owner(
         """
         path = utils.check_path(path)
         version = utils.check_version(version)
+        path_with_version = self._path_with_version(path, version)
 
         return utils.call_function_on_backend(
             self._owner,
-            path,
-            version,
+            path_with_version,
         )
 
+    def _path_with_version(
+            self,
+            path: str,
+            version: str,
+    ) -> str:
+        r"""Convert to versioned path.
+
+        <root>/<name>
+        ->
+        <root>/<version>/<name>
+
```

Comment:
Would be nice, if you could add a line here to the docstring also documenting how the returned path looks if `self._legacy_file_structure` is `True`.

Reply:
done